python/manylinux1/build_arrow.sh

```diff
@@ -95,6 +94,7 @@ cmake -DCMAKE_BUILD_TYPE=Release \
     -DARROW_TENSORFLOW=ON \
     -DARROW_ORC=ON \
     -DARROW_WITH_BZ2=ON \
+    -DARROW_WITH_ZLIB=ON \
```

Comment:
I think that's already implied?

Reply:
Yep, but we should prefer explicit cmake options IMO.

Reply:
Other compression options aren't listed here.

Reply:
Ok, removing it :)